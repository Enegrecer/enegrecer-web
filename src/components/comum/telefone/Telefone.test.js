import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Telefone from './../telefone';

describe('Telefone', () => {
  it('deve seguir a estrutura definida ', () => {
    const handleChange = jest.fn();
    const telefone = shallow(<Telefone
      id="telefone"
      handleChange={handleChange}
      label={'Telefone'}
      value={''}
    />);
    const telefoneJson = toJson(telefone);
    expect(telefoneJson).toMatchSnapshot();
  });
});

