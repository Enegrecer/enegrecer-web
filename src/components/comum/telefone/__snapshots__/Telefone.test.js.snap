// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Telefone deve seguir a estrutura definida  1`] = `
<div
  className="input-field col s6"
>
  <InputElement
    id="telefone"
    mask="(99) 99999-9999"
    maskChar=" "
    name="telefone"
    onChange={[Function]}
    type="text"
    value=""
  />
  <label
    htmlFor="telefone"
  >
    Telefone
  </label>
</div>
`;
